sns-book/src/domain-name/domain-find-for-owner.md

```diff
@@ -1,6 +1,28 @@
 # Get all domains of a user
 
-You can retrieve all the domain names of a user with the following `MemcmpFilter`
+If you are using `@bonfida/spl-name-service` you can use the following code:
+
+```js
+import { getAllDomains } from "@bonfida/spl-name-service";
+
+// ...
+
+const domains = await getAllDomains(connection, user);
+```
+
+The function above will return an array of 2 Public Keys. Use the toBase58() method to convert the keys into a human readable format.
+
+```js
+domains.map((domain) => domain.toBase58());
+```
+
+Another option to retrieve not only human readable Public Keys, but also the domain name in a human readable string format is to use the following function from spl-name-service library below.
+
+```js
+const domainKeysWithStrings = getDomainKeysWithReverses(connection, user);
```

Comment:
Variable name `domainKeysWithStrings` is a bit misleading

Reply:
As you mentioned above, reverses is known to be the human readable format so I think domainsWithReverses will be a better variable name.